src/linker/Linker.Steps/CodeRewriterStep.cs

```diff
@@ -137,7 +137,7 @@ MethodBody CreateThrowLinkedAwayBody (MethodDefinition method)
 			ctor = Context.MarkedKnownMembers.NotSupportedExceptionCtorString;
 			ctor = assembly.MainModule.ImportReference (ctor);
 
-			il.Emit (OpCodes.Ldstr, "Linked away");
+			il.Emit (OpCodes.Ldstr, "Linked away: " + method.FullName);
```

Comment:
This should be controlled by an option, since it's going to add a lot of extra strings to the assembly.

Reply:
How do I add an option and what should it be called: `indetify-linked-away`?